### PR TITLE
fix(blog): agility - fix local links

### PR DESCRIPTION
**Pull request by muescha**
## Description

- fix links to localhost in to code blocks

## Related Issues

- #22985 `Add blog post Agility CMS upgrades to Gatsby with author avatar`

### CODE REVIEW

File: docs/blog/2020-04-13-upgrading-to-jamstack-with-agility/index.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Gatsby will pull down all the content for our website and put it into GraphQL. T
 
 We need to make our new website look just like the old one, so we need to match the colors, fonts, and other visual styles as much as we can. You may want to pull in the CSS from your old site—or start from scratch if you want to make a clean break.
 
-Either way, this is a great time to familiarize yourself with the GraphQL data in your website. Point your browser to [http://localhost:8000/\_\_\_graphql](http://localhost:8000/___graphql) to start exploring your data, and you can build the query access and Shared Content or Sitemap data. You can see that all content is available, grouped by content definition name.
+Either way, this is a great time to familiarize yourself with the GraphQL data in your website. Point your browser to `http://localhost:8000/___graphql` to start exploring your data, and you can build the query access and Shared Content or Sitemap data. You can see that all content is available, grouped by content definition name.
```
Comment (vladar) [inline docs/blog/2020-04-13-upgrading-to-jamstack-with-agility/index.md:120]: This will remove the link, right? What is the motivation behind this change?

Comment (muescha) [reply docs/blog/2020-04-13-upgrading-to-jamstack-with-agility/index.md:120]: yes

per style guide this links should go into code blocks, so there is no 404 while not running a local instance
